.github/workflows/samples.yml

```
@@ -0,0 +1,64 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Workflow for running Examples of IREE usage against releases periodically.
+
+name: samples
+
+on:
+  schedule:
+    - cron: "0 13 * * *"
+  workflow_dispatch:
+  # DO NOT SUBMIT: Turn off presubmit build
+  pull_request:
+
+jobs:
+  colab:
+    runs-on: ubuntu-20.04-64core
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
+      - name: "Testing Colab Notebooks"
+        run: |
+          ./build_tools/github_actions/docker_run.sh  \
+            gcr.io/iree-oss/samples@sha256:291d87d125b41e1539dbb96c7c4f734cc98fc158a36d18496cda360658f00bde \
+            ./samples/colab/test_notebooks.py
+
+  samples:
+    runs-on: ubuntu-20.04-64core
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
+      - name: "Testing Samples"
+        run: |
+          ./build_tools/github_actions/docker_run.sh  \
+            gcr.io/iree-oss/samples@sha256:291d87d125b41e1539dbb96c7c4f734cc98fc158a36d18496cda360658f00bde \
+            ./build_tools/testing/test_samples.sh
+
+  # FIXME: unify this with host build from ci.yml. This is complicated by
+  # incompatibility between storage options for GitHubmanaged and self-hosted
+  # runners.
+  wasm:
```

This will include WebGPU and WebAssembly code, so maybe rename to just "web"
```suggestion
  web:
```